bee-node/bee-node/src/cli.rs

```diff
@@ -47,4 +51,14 @@ impl NodeCliArgs {
     pub fn version(&self) -> bool {
         self.version
     }
+
+    /// Returns the listening port for gossip.
+    pub fn gossip_port(&self) -> Option<&u16> {
+        self.gossip_port.as_ref()
+    }
+
+    /// Returns the identity of the node if specified.
```

Comment:
Why do we say "if specified" here but not on the port ? I wouldn't mention it as it's an Option tbh